yaki_admin/CONTRIBUTING.md

```diff
@@ -0,0 +1,240 @@
+
+# XPEHO Internal contributions
+
+## Accepted contributors
+
+XPEHO company members are the only ones able to make code changes.
+
+## Contribution process
+
+1. Fill an issue to explain the change to do
+
+2. XPEHO team will make some triage
+
+3. Issue is accepted or rejected
+
+4. Accepted issues will be prioritized
+
+5. XPEHO Dev team will plan issue modifications in future dev cycle
+
+6. Modifications will be pushed on a dedicated branch
+
+7. Once ready, a Pull request is opened for code review and CI checks
+
+8. Checked modification will be merged to the main branch
+
+## Git flow
+
+We work with `single trunk based developpment`
+
+More information at [https://trunkbaseddevelopment.com/](https://trunkbaseddevelopment.com/)
+
+### Branch names
+
+All branches must reach the following convention
+
+```
+<prefix>/<change_name>
+```
+
+**Example**
+
+```
+feat/authentication
+```
+
+**Accepted prefix list**
+
+- feat
+- feature
+- fix
+- hotfix
+- docs
+- style
+- refactor
+- chore
+- test
+
+**Prefix list explaination**
+
+**feat**
+
+Branch containing small enhancement changes
+
+**feature**
+
+Branch containing multiple enhancement changes from multiple feat ones
+
+**fix**
+
+Branch containing only a bug fix
+
+**hotfix**
+
+Branch containing only a PROD hurry fix
+
+**docs**
+
+Branch containing only documentation changes
+
+**style**
+
+Branch containing only UI style changes
+
+**refactor**
+
+Branch containing breaking changes
+
+**chore**
+
+Branch containing tooling change like CI/CD, Issue templates, gitignore file etc...
+
+**test**
+
+Branch containing test
+
+### Commit convention
+
+We respect the `Angular Js Commit message` convention
+
+Full documentation at [https://gist.github.com/stephenparish/9941e89d80e2bc58a153](https://gist.github.com/stephenparish/9941e89d80e2bc58a153)
+
+The commit message must respect the following convention
+
+```
+<type>(<scope>): <subject>
+```
+
+**Example**
+
+```
+feat(authentication): Create authentication screen
+```
+
+**Accepted prefix list**
+
+Accepted prefix list is the same as the branch prefix list
+
+- feat
+- fix
+- hotfix
+- docs
+- style
+- refactor
+- chore
+- test
+
+## Pull Requests
+
+Each change must be proposed to merge using a `Pull Request` from the current branch to the main one
+
+For more information about `Pull Request` creation please visit [https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request)
+
+## Naming Convention
+
+For this project, naming convention for all language use in this project should be on lower camelCase,
+except for Classes, enum types, typedefs, and type parameters these one should be on kebab-case.
```

Comment:
Only component reference in the template should be on kebab-case everything else should be on PascalCase